pkg/snapstore/abs_snapstore.go

```diff
@@ -72,35 +74,41 @@ func NewABSSnapStore(config *brtypes.SnapstoreConfig) (*ABSSnapStore, error) {
 		Retry: azblob.RetryOptions{
 			TryTimeout: downloadTimeout,
 		}})
-	u, err := url.Parse(fmt.Sprintf("https://%s.%s", storageAccount, brtypes.AzureBlobStorageHostName))
+
+	absURI := brtypes.AzureBlobStorageGlobalURI
```

Comment:
what is `absURI`? Should i interpret it as absolute URI or azure blob store URI?

Reply:
I understand your point of view, will be removing this altogether - and will make use of the `storageAPIEndpoint` throughout.